### PR TITLE
String tweaks

### PR DESCRIPTION
**Pull request by qaisjp**
I've tweaked a few strings here and there.

### CODE REVIEW

File: applications/forms.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def clean_code_conduct(self):
         # self.instance.pk is None if there's no Application existing before
         # https://stackoverflow.com/questions/9704067/test-if-django-modelform-has-instance
         if not cc and not self.instance.pk:
-            raise forms.ValidationError("In order to apply and attend you have to accept our code of conduct")
+            raise forms.ValidationError("You must accept our code of conduct")
```
Comment (casassg) [inline applications/forms.py]: Any specific reason to change the language here? I think "must" sounds a bit tough 😅

File: applications/forms.py [hunk 2]
```diff
@@ -201,8 +201,8 @@ class Meta:
         }
 
         labels = {
-            'gender': 'What gender do you associate with?',
-            'graduation_year': 'What year are you graduating?',
+            'gender': 'What gender do you identify as?',
```
Comment (casassg) [inline applications/forms.py:205]: Good phrasing 👍

**Review by casassg — COMMENTED** [review]
Looks good, except for the "must" part. I'm not sure about that one.

File: applications/forms.py (hunk 1 above)
Comment (qaisjp) [reply applications/forms.py]: How about `To attend %s you must abide by our code of conduct` instead? Still uses `must` but I think lightened a little by the context?

Comment (casassg) [reply applications/forms.py]: Sounds better 👍

**Review by casassg — APPROVED** [review]
lgtm

**Pull request merged**